plugins/woocommerce/client/admin/client/settings-payments/onboarding/providers/woopayments/steps/payment-methods-selection/index.tsx

```diff
@@ -0,0 +1,141 @@
+/**
+ * External dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
+import { RecommendedPaymentMethod } from '@woocommerce/data';
+import { useState, useEffect } from '@wordpress/element';
+import apiFetch from '@wordpress/api-fetch';
+
+/**
+ * Internal dependencies
+ */
+import { useOnboardingContext } from '../../data/onboarding-context';
+import { PaymentMethodListItem } from '~/settings-payments/components/payment-method-list-item';
+import './payment-methods-selection.scss';
```

Comment:
Nitpick: I think we can rename this file to styles.scss, since there are no other style files in that directory.

Reply:
Indeed, renamed in [d0f2ef4](https://github.com/woocommerce/woocommerce/pull/56554/commits/d0f2ef47510051821a6f5e86335aa1c1a1aae39d)